internal/faq_templates/a4code.txt

```diff
@@ -0,0 +1,54 @@
+What is a4code?
+===
+[b]a4code[/b] is a lightweight markup language used on this site for formatting text. It uses square brackets to denote formatting tags.
```

Comment:
This code is invalid @jules it's not correct. Please refer to the a4code2html if there is anything else that is inconsistent that implied this form please update that too